### PR TITLE
Update config.toml for release-1.20 branch.

### PR DESCRIPTION
**Pull request by PI-Victor**
This commit will bump all the versions according to the latest
release patches.

/assign @jimangel @irvifa @sftim 

Signed-off-by: Victor Palade <victor@cloudflavor.io>

/hold till release day

~**p.s.: The branch `release-1.21` referenced in this PR will be created on release day.**~
release-1.21 was created!

/cc @reylejano

### CODE REVIEW

**Comment by reylejano** [comment]
For reference, [latest patch releases](https://github.com/kubernetes/website/blob/main/content/en/releases/patch-releases.md)

@PI-Victor , leave the configs for 1.18
https://kubernetes.io/docs/home/supported-doc-versions/ explains the documentation policy: documentation for the current version of Kubernetes and the four previous versions of Kubernetes.

File: config.toml [hunk 1]
```diff
@@ -177,42 +177,42 @@ js = [
...
 url = "https://v1-19.docs.kubernetes.io"
 
 [[params.versions]]
-fullversion = "v1.18.17"
+fullversion = "v1.18.20"
 version = "v1.18"
-githubbranch = "v1.18.17"
+githubbranch = "v1.18.20"
 docsbranch = "release-1.18"
 url = "https://v1-18.docs.kubernetes.io"
```
Comment (PI-Victor) [inline config.toml:215]: 1.18.20 version this is now added back

**Comment by reylejano** [comment]
looks good
/approve

cancel the hold when ready

**Pull request merged**